src/modules/admin/components/forms/FormDefinitionActionsCard.tsx

```diff
@@ -0,0 +1,100 @@
+import { Divider, Stack, Typography } from '@mui/material';
+
+import React from 'react';
+import { generatePath } from 'react-router-dom';
+import ButtonLink from '@/components/elements/ButtonLink';
+import { CommonCard } from '@/components/elements/CommonCard';
+
+import DuplicateFormButton from '@/modules/admin/components/forms/DuplicateFormButton';
+import EditFormButton from '@/modules/admin/components/forms/EditFormButton';
+import {
+  formatRelativeDateTime,
+  parseHmisDateString,
+} from '@/modules/hmis/hmisUtil';
+import { RootPermissionsFilter } from '@/modules/permissions/PermissionsFilters';
+import { AdminDashboardRoutes } from '@/routes/routes';
+import { FormIdentifierDetailsFragment, FormStatus } from '@/types/gqlTypes';
+
+interface Props {
+  formIdentifier: FormIdentifierDetailsFragment;
+}
+
+const FormDefinitionActionsCard: React.FC<Props> = ({ formIdentifier }) => {
```

Comment:
Just refactored this out of `src/modules/admin/components/forms/FormDefinitionDetailPage.tsx` and added the `DuplicateFormButton` logic